prow/plugins/releasenote/releasenote.go

```diff
@@ -123,25 +131,29 @@ func handleComment(gc githubClient, log *logrus.Entry, ic github.IssueCommentEve
 	isAuthor := ic.Issue.IsAuthor(ic.Comment.User.Login)
 
 	if !isMember && !isAuthor {
-		resp := "you can only set release notes if you are the author or an org member."
-		log.Infof("Commenting with \"%s\".", resp)
+		format := "you can only set the release note label to %s if you are the PR author or an org member."
+		resp := fmt.Sprintf(format, releaseNoteNone)
 		return gc.CreateComment(org, repo, number, plugins.FormatICResponse(ic.Comment, resp))
 	}
 
-	// Add the requested label if necessary.
-	var errs []error
-	if !ic.Issue.HasLabel(nl) {
-		log.Infof("Adding %s label.", nl)
-		if err := gc.AddLabel(org, repo, number, nl); err != nil {
-			errs = append(errs, err)
+	// Add the releaseNoteNone label if the release-note block is none or empty.
```

Comment:
Is this comment correct? It looks like it's adding a comment if there's release note or releasenoteactionrequired?

Reply:
Its correct. I changed it to 
"Don't allow the /release-note-none command if the release-note block contains a valid release note."
to be more clear.